webapp/components/test-results-chart.html

```diff
@@ -157,41 +144,33 @@
         this.showForNow = false;
       }
 
-      // Load results of `tests` from `chunkSize` runs labeled with `labels`
-      // from prior to `to` iff `show`.
-      async loadNext(tests, chunkSize, maxRuns, labels, to, show) {
-        if (!show || tests.length === 0 || this.values.length >= maxRuns) {
+      computeTestRunQueryParams(shas, aligned, master, labels, productSpecs, to, from, maxCount) {
+        maxCount = this.chunkSize;
+        return super.computeTestRunQueryParams(shas, aligned, master, labels, productSpecs, to, from, maxCount);
+      }
+
+      // eslint-disable-next-line no-unused-vars
+      loadResults(tests, query) {
```

Comment:
Any benefits to stop using async/await here and`loadNext`? `await` looks clearer to me.

Reply:
More obvious that it's returning a chainable promise, but could be swayed to keep await and leave understanding as an exercise for the reader.

Reply:
I feel the opposite: without type annotation (of the return value), `async` makes it more obvious that the function returns a promise.

Reply:
Reverted.